### DISCUSSION TITLE
Copilot X vsc side window not conversational

### DISCUSSION THREAD

**Discussion by laviRZ** [community/community#58180] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Even though it's designed to look that way, the copilot side window in vsc is not a continuous conversation, and copilot has no awareness of previous messages. This can prove to be quite annoying since you can't follow up on questions.
Example:
![image](https://github.com/community/community/assets/29277997/9af9329d-013b-4537-814e-ddc679b6085b)

**Comment by Akash1134** [comment, marked as answer]
Hi @laviRZ ,

We are aware of this and the feedback related to this functionality is being tracked in this [discussion](https://github.com/microsoft/vscode-copilot-release/issues/74).  Thanks!
